sections/preliminaries.tex

```diff
@@ -0,0 +1,136 @@
+\section{Preliminaries}\label{sec:preliminaries}
+This section covers previous work within topic modelling that will be expanded upon in this paper.
+
+\subsection{\acrlong{lda}}
+The purpose of \gls{lda}, and topic models in general, is to create a tool for exploring collections of text.
+Topic models do this by uncovering the underlying semantic structure of a text collection by using hierarchical Bayesian models.
+\Gls{lda} uncovers this semantic structure by discovering patterns of word use in documents and finding topics based on these~\cite{blei2009topic}.
+
+The standard \gls{lda} by \citet{blei2003latent} can be described by the following generative process, which is the way the model assumes the documents arose:
```

Comment:
we dont actually begin with the generative process when we say we do, we just explain variables